containers/README.md

```diff
@@ -8,17 +8,34 @@ This folder holds source code and Docker configurations for the container images
 
 This section describes the process for building and pushing container images to Artifact Registry, where they will later be used to deploy to a container runtime environment like GKE.
 
-1. Navigate to the directory of the container image you are building (eg: the `ffmpeg` container image):
+1. Run this command to invoke Cloud Build to build and push your container image to Artifact Registry
 
     ```bash
-    cd ./ffmpeg
+    export APP_NAME=ffmpeg-api
+    export COMMIT_SHA7=$(echo $(git rev-parse HEAD) | cut -c1-7)
+
+    gcloud builds submit --config ./containers/$APP_NAME/cloudbuild.yaml \
+      --region us-central1 \
+      --substitutions _SKAFFOLD_MODULE=$APP_NAME,_RELEASE_ID=$COMMIT_SHA7
     ```
 
-2. Run this command to invoke Cloud Build and build and deploy your container image to Artifact Registry
+## Deploy Container Images
+
+During the Cloud Build, the last step will deploy the latest container to Google Cloud Deploy. The release is named `rel-$COMMIT_SHA7`
 
     ```bash
-    export PROJECT_ID=$(gcloud config get-value core/project)
-    gcloud builds submit --config ./cloudbuild.yaml \
-        --region us-central1 \
-        --substitutions _PROJECT_ID=$PROJECT_ID
+    gcloud deploy releases create rel-${_RELEASE_ID} \
+      --region=$LOCATION \
+      --delivery-pipeline=$_SKAFFOLD_MODULE-cd \
+      --skaffold-version="2.13" \
+      --skaffold-file=./containers/$_SKAFFOLD_MODULE/skaffold.yaml \
+      --build-artifacts=/workspace/artifacts.json
     ```
+
+On successful completion of Cloud Build, the application will automatically be promoted to the `dev` Target. From there navigate to the Cloud Deploy UI, to manual promot and approve rollouts to `staging` and `prod` Targets.
```

Comment:
*manually promote
s/Targets/targets/g

Reply:
It is **automatically** deployed to `dev` but there after manually promote to stage and prod

I did meaningfully capitalize **T** here as reference to Cloud Deploy `Target` as a resource

Reply:
Definitely put the capital T back if that's the case, I thought it was just a typo.